examples/click-events-basic/src/index.js

```diff
@@ -36,10 +37,11 @@ chart.addEventListener("click", (payload) => {
   }
 
   document.getElementById("info").innerHTML = "<ul>" + infoText + "</ul>";
-});
```

Comment:
Interesting, does this mean the code didn't work before?

Reply:
Ah, I see. Ignore me

Reply:
I think it just looks strange in Github.
The round bracket is the closing bracket to the addEventListener which was moved further down. And the new curly bracket is closing the clickHandler function.